server/src/test/scala/org/http4s/server/middleware/ErrorActionSuite.scala

```diff
@@ -18,16 +18,15 @@ package org.http4s.server.middleware
 
 import cats.effect.IO
 import cats.effect.concurrent.Ref
+import com.comcast.ip4s.{IpAddress, Port, SocketAddress}
 import org.http4s._
 import org.http4s.Request.Connection
-import org.http4s.syntax.all._
 import org.http4s.dsl.io._
+import org.http4s.syntax.all._
 import org.typelevel.vault.Vault
 
-import java.net.{InetAddress, InetSocketAddress}
-
 class ErrorActionSuite extends Http4sSuite {
-  val remote = "192.168.0.1"
+  val remote = IpAddress("192.168.0.1").get
```

Comment:
We're running up against a problem here where our `ipv4` and `ipv6` interpolators conflict with ip4s', and ip4s' are imported at the root level, so they're almost always going to win.  I think we should retire ours.  Most of what @hamnis just did for ours could continue on: just remove the conflicting ones.